### PR TITLE
Feature/blocking buttons when blocked mode

### PR DESCRIPTION
**Pull request by JanBan**
It is now unable to toggle blocking while in blocked mode or resetting blocked mode while it is on,

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -48,29 +48,46 @@ chrome.browserAction.onClicked.addListener(function toggleBlocking(){
 			});
 		}
 		else{
-			var now = new Date().getTime();
-			var timeLeft = data.timerData.blockUntilMilliseconds - now;
-			var minutes = Math.floor((timeLeft % (1000 * 60 * 60)) / (1000 * 60));
-			var seconds = Math.floor((timeLeft % (1000 * 60)) / 1000);
-			alert("Timer mode enabled! " + minutes + " minutes " + seconds + " seconds left.");
+			if(updateTimer(data.timerData) == false){
```
Comment (SuperJarek) [inline background.js]: "!" instead of "==false"

File: background.js [hunk 2]
```diff
@@ -48,29 +48,46 @@ chrome.browserAction.onClicked.addListener(function toggleBlocking(){
...
+function updateTimer(timerData){
+	let timeLeft = timerData.blockUntilMilliseconds - Date.now();
+	if (timeLeft <= 0){  //unblock
+		timerData.isTimerEnabled = false;
+		chrome.storage.sync.set({'timerData': timerData}, function() {
+			turnFilteringOff();
+		});
+		return true;
+	}
+	else{
```
Comment (SuperJarek) [inline background.js]: Else is unnecessary

File: background.js [hunk 3]
```diff
@@ -48,29 +48,46 @@ chrome.browserAction.onClicked.addListener(function toggleBlocking(){
...
-					let timeLeft = data.timerData.blockUntilMilliseconds - Date.now();
-					if (timeLeft <= 0){  //unblock
-						data.timerData.isTimerEnabled = false;
-						chrome.storage.sync.set({'timerData': data.timerData}, function() {
-							turnFilteringOff();
-						});
-						return;
-					}
-					else{
+					if(updateTimer(data.timerData) == false){
```
Comment (SuperJarek) [inline background.js]: "!" instead of "== false"

File: blockedModeSetup.js [hunk 4]
```diff
@@ -2,6 +2,12 @@ const ENTER_KEY_CODE = 13;
 
 document.addEventListener('DOMContentLoaded', function() {
 	let textField = document.getElementById('duration');
+    let startButton = document.getElementById('start');
```
Comment (SuperJarek) [inline blockedModeSetup.js]: On more tab here is needed?

**Review by SuperJarek — CHANGES_REQUESTED** [review]

File: blockedModeSetup.js (hunk 4 above)
Comment (JanBan) [reply blockedModeSetup.js]: To github coś popsuł znów chyba, "u mnie działa". Może z kodowaniem dokumentu w n++ musimy poeksperymentować

**Pull request merged**